recipe/passwordless/smsdelivery/twilioService/passwordlessLogin.go

```diff
@@ -0,0 +1,28 @@
+package twilioService
+
+import (
+	"fmt"
+
+	"github.com/supertokens/supertokens-golang/ingredients/smsdelivery"
+)
+
+func getPasswordlessLoginSmsContent(input smsdelivery.PasswordlessLoginType) smsdelivery.TwilioGetContentResult {
+	return smsdelivery.TwilioGetContentResult{
+		Body:          getPasswordlessLoginSmsBody(input.CodeLifetime, input.UrlWithLinkCode, input.UserInputCode),
+		ToPhoneNumber: input.PhoneNumber,
+	}
+}
+
+func getPasswordlessLoginSmsBody(codeLifetime uint64, urlWithLinkCode *string, userInputCode *string) string {
+	var message string = ""
+
+	if urlWithLinkCode != nil && userInputCode != nil {
+		message = `Enter OTP: ` + *userInputCode + ` OR click this link: ` + *urlWithLinkCode + ` to login.`
+	} else if urlWithLinkCode != nil {
+		message = `Click this link: ` + *urlWithLinkCode + ` to login.`
+	} else {
+		message = `Enter OTP: ` + *userInputCode + ` to login.`
+	}
+	message += ` It will expire in ` + fmt.Sprint(codeLifetime) + ` seconds.`
```

Comment:
I don't think code lifetime is in seconds.

Reply:
<img width="480" alt="image" src="https://user-images.githubusercontent.com/650429/170461804-519f842c-1ac1-4406-bfe4-9592950bab37.png">
It's mentioned as seconds in the node as well. Is it different for golang?

Reply:
Have add Humanise functions and also added tests for it. I have also corrected passwordless login email body to use this.